plugins/tech-radar/src/components/Radar/utils.ts

```diff
@@ -81,30 +81,33 @@ export const adjustQuadrants = (
     },
   ];
 
-  quadrants.forEach((quadrant, index) => {
+  return quadrants.slice().map((quadrant, index) => {
     const legendParam = legendParams[index % 4];
 
-    quadrant.index = index;
-    quadrant.radialMin = (index * Math.PI) / 2;
-    quadrant.radialMax = ((index + 1) * Math.PI) / 2;
-    quadrant.offsetX = index % 4 === 0 || index % 4 === 3 ? 1 : -1;
-    quadrant.offsetY = index % 4 === 0 || index % 4 === 1 ? 1 : -1;
-    quadrant.legendX = legendParam.x;
-    quadrant.legendY = legendParam.y;
-    quadrant.legendWidth = legendParam.width;
-    quadrant.legendHeight = legendParam.height;
+    return {
+      ...quadrant,
+      index,
+      radialMin: (index * Math.PI) / 2,
+      radialMax: ((index + 1) * Math.PI) / 2,
+      offsetX: index % 4 === 0 || index % 4 === 3 ? 1 : -1,
+      offsetY: index % 4 === 0 || index % 4 === 1 ? 1 : -1,
+      legendX: legendParam.x,
+      legendY: legendParam.y,
+      legendWidth: legendParam.width,
+      legendHeight: legendParam.height,
+    };
   });
 };
 
 export const adjustEntries = (
-  entries: Entry[],
+  _entries: Entry[],
```

Comment:
`_` is used to indicate an unused parameter. Stick to `entries` and use a different name for the internal var instead